project1/client/client_tasks/upload_task.h

```diff
@@ -12,19 +12,18 @@
 namespace client_tasks {
 class UploadTask : public thread_pool::Task {
  public:
-  UploadTask(int client_fd) {
-    client_fd_ = client_fd;
-  }
+  UploadTask(int client_fd);
```

Comment:
...and this one. Also, this should probably be marked `explicit`? I thought CLion usually yelled at you if you forgot to do that.